validator/keymanager/remote-web3signer/keymanager.go

```diff
@@ -38,54 +53,323 @@ type SetupConfig struct {
 	// Either URL or keylist must be set.
 	// a static list of public keys to be passed by the user to determine what accounts should sign.
 	// This will provide a layer of safety against slashing if the web3signer is shared across validators.
-	ProvidedPublicKeys [][48]byte
+	ProvidedPublicKeys []string
 }
 
 // Keymanager defines the web3signer keymanager.
 type Keymanager struct {
 	client                internal.HttpSignerClient
 	genesisValidatorsRoot []byte
-	publicKeysURL         string
-	providedPublicKeys    [][48]byte
+	providedPublicKeys    [][48]byte          // source of truth flag loaded + file loaded + api loaded keys
```

Comment:
```suggestion
	providedPublicKeys    [][48]byte          // (source of truth) flag loaded + file loaded + api loaded keys
```
This makes the comment slightly more readable IMO